test/parallel/test-zlib-failed-init.js

```diff
@@ -8,9 +8,11 @@ const zlib = require('zlib');
 // For raw deflate encoding, requests for 256-byte windows are rejected as
 // invalid by zlib.
 // (http://zlib.net/manual.html#Advanced)
-assert.throws(() => {
-  zlib.createDeflateRaw({ windowBits: 8 });
-}, /^Error: Init error$/);
+if (process.versions.zlib.match(/^\d+\.\d+\.[9]|\d{2,}$/)) {
```

Comment:
I think it would be good to include a link to the reasoning in zlib for this directly rather than having people `blame` and dig the link from the PR - but I'm fine with other way.